Dockerfile

```diff
@@ -3,74 +3,100 @@
 FROM centos:centos7
 MAINTAINER perfSONAR <perfsonar-user@perfsonar.net>
 
-
-RUN yum -y install epel-release
-RUN yum -y install http://software.internet2.edu/rpms/el7/x86_64/main/RPMS/perfSONAR-repo-0.8-1.noarch.rpm 
-RUN yum -y update; yum clean all
-RUN yum -y install perfsonar-testpoint
-RUN yum -y install supervisor rsyslog net-tools sysstat iproute bind-utils tcpdump # grab a few other needed tools
```

Comment:
removed the grab other needed tools, actually required tools will be handled by the rpm bundle and don't need to duplicate that for confusion
supervisor and rsyslog are left included since there's setup specific information included in the Dockerfile